### PR TITLE
Initial approach

### PR DESCRIPTION
**Pull request by lnmedrano**
## Summary
- Compare schema in pre-commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+At last, if no breaking changes are found, it replace the old schema with the new schema.
+## Usage
+It must be called with: `node schema_inspector.js <oldSchemaPath> <newSchemaPath>`.
+
+Supported extensions: 
+* oldSchemaPath:
+  * .graphql
+  * .json
+* newSchemaPath:
+  * .js file that exports a GraphQLType variable named schema.
```
Comment (mpicollo-wolox) [inline README.md]: Some things here:
- Add MIT license 
- How contribute to this
- Some instructions for local running
- Mention Wolox as author and maintainer

File: schema-inspector.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+};
+
+try {
+  if (fs.existsSync(oldSchemaFile)) {
+    console.log(info, 'Schema inspection:');
+    const changes = diff(buildSchema(importSchema(oldSchemaFile)), buildSchema(newSchema));
+    changes.forEach(change => {
+      console.log(criticalitySymbols[change.criticality.level], change.message);
+    });
+    if (changes.some(c => c.criticality.level === breaking)) {
```
Comment (mpicollo-wolox) [inline schema-inspector.js]: Add some flag for bypass breaking changes in schema (like level)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "graphql-inspector-wolox-node",
+  "version": "0.1",
```
Comment (mpicollo-wolox) [inline package.json]: Use semantic version here,`1.0.0` for initial version

File: package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "description": "",
+  "main": "schema-inspector.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```
Comment (mpicollo-wolox) [inline package.json]: author is `Wolox` check other packages

File: schema-inspector.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
```
Comment (mpicollo-wolox) [inline schema-inspector.js]: Normally packages use an index like this
```
module.exports = require('./lib')
```
Lib directory contains source code and exports in an index file

**Review by picollomartin — CHANGES_REQUESTED** [review]
Amazing job 🎉 , some comments and two missing things: 
- Changelog of versions
- package-lock.json

File: lib/index.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+const processcwd = process.cwd();
+const oldSchemaFile = `${processcwd}/${args[OLD_SCHEMA]}`;
+const { schema } = require(`${processcwd}/${args[NEW_SCHEMA]}`);
+const newSchema = printSchema(schema);
+
+try {
+  logger.info(info, 'Schema inspection:');
+  let changes = [];
+  if (fs.existsSync(oldSchemaFile)) {
+    changes = getChangesInSchema(oldSchemaFile, newSchema);
```
Comment (picollomartin) [inline lib/index.js]: You could use const in changes and do here something like `changes.push(...getChangesInSchema(oldSchemaFile, newSchema))`

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by mpuricawolox — DISMISSED** [review]

**Review by picollomartin — DISMISSED** [review]

**Review by mpuricawolox — APPROVED** [review]

**Pull request merged**